### PR TITLE
Implemented retry of authentication

### PR DESCRIPTION
**Pull request by jobec**
Fix for issue #7

### CODE REVIEW

File: infoblox_netmri/__init__.py [hunk 1]
```diff
@@ -76,6 +80,37 @@ def __init__(self, host, username, password, api_version="auto",
...
+            method (str): Any of "get", "post", "delete"
+            data (any): Possible extra data to send with the request
+            extra_headers (dict): Possible extra headers to send along in the request
+        Returns:
+            dict
+        """
+        retry = True
+        attempts = 0
+
+        while retry and attempts < 1:
```
Comment (johnbelamaric) [inline infoblox_netmri/__init__.py]: You don't really need retry. If the request is successful, you return. If not, you either retry, or you raise. So, retry never gets set to false.

Comment (jobec) [inline infoblox_netmri/__init__.py]: That's right, good catch. I'll fix it.

It must have sneaked in there from my previous version without a loop.

**Comment by jobec** [comment]
Any plans on releasing a new version with this pull request in it?

**Comment by jobec** [comment]
5 months later: Any plans on releasing a new version?

**Comment by bondar-pavel** [comment]
@johnbelamaric ping :)

**Comment by johnbelamaric** [comment]
Done!

**Comment by bondar-pavel** [comment]
Thanks!

**Pull request merged**